exports/dataset.cpp

```diff
@@ -624,9 +626,47 @@ PYBIND11_MODULE(dataset, m) {
           "numpy", &as_py_array_t_variant<Variable, double, float, int64_t,
                                           int32_t, char, bool>)
       .def_property_readonly("data", &as_VariableView::get<Variable>)
+      .def(py::self + py::self, py::call_guard<py::gil_scoped_release>())
+      .def(py::self + float(), py::call_guard<py::gil_scoped_release>())
+      .def("__add__",
+           [](const Variable &a, const VariableSlice &b) { return a + b; },
+           py::is_operator())
+      .def(py::self - py::self, py::call_guard<py::gil_scoped_release>())
+      .def(py::self - float(), py::call_guard<py::gil_scoped_release>())
+      .def("__sub__",
+           [](const Variable &a, const VariableSlice &b) { return a - b; },
+           py::is_operator())
+      .def(py::self * py::self, py::call_guard<py::gil_scoped_release>())
+      .def(py::self * float(), py::call_guard<py::gil_scoped_release>())
+      .def("__mul__",
+           [](const Variable &a, const VariableSlice &b) { return a * b; },
+           py::is_operator())
+      .def(py::self / py::self, py::call_guard<py::gil_scoped_release>())
+      .def(py::self / float(), py::call_guard<py::gil_scoped_release>())
+      .def("__truediv__",
```

Comment:
We are not actually doing "true division". Should this be `__floordiv__`? If so, will `var1/var2` still work and use `__floordiv__`?

Reply:
```from dataset import * 
import numpy as np 
v1 = Variable(Data.Value, [Dim.X], np.array([3.0]*10)) 
v2 = Variable(Data.Value, [Dim.X], np.array([2.0]*10)) 
(v1/v2).numpy 
```
True division would be expected in the above (yes either v1 or v2 as slice, but just for illustration). Why floor division?

Reply:
We do not have true division implemented in C++, so the current naming is misleading. I am not sure where we want to support true division, and don't think we should decide this at this point.

You example contains floats not integers, so the issue does not arise.